src/script/lua_api/l_env.h

```diff
@@ -326,6 +329,34 @@ class LuaRaycast : public ModApiBase
 	static const char className[];
 };
 
+class Lua3DLine : public ModApiBase
+{
+private:
+	u32 m_id;
+	bool markedForRemoval;
```

Comment:
`m_` prefix and snake case.
Suggested name change: `m_pending_removal` (same as in SAO).